Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2114,16 +2114,11 @@ public func matmul<Scalar: Numeric>(
     _ rhs: Tensor<Scalar>,
     transposed transposeB: Bool = false
 ) -> Tensor<Scalar> {
-    switch (lhs.rank, rhs.rank) {
-    case (3..., 3...):
-        return Raw.batchMatMulV2(lhs, rhs, adjX: transposeA, adjY: transposeB)
-    case (2, 3...):
-        return Raw.batchMatMulV2(lhs.expandingShape(at: 1), rhs, adjX: transposeA, adjY: transposeB)
-    case (3..., 2):
-        return Raw.batchMatMulV2(lhs, rhs.expandingShape(at: 1), adjX: transposeA, adjY: transposeB)
-    default:
-        return Raw.matMul(lhs, rhs, transposeA: transposeA, transposeB: transposeB)
+    if lhs.rank > 2 || rhs.rank > 2 {
+      // TODO: Conjugate to make compatible with the adjoint.
```

Comment:
File a bug for this?

Reply:
I'll fix this as part of this PR. Give me a few minutes.

Reply:
Actually, nvm this only matters if we support complex number types. Should I clarify this in the TODO alone or also file a bug?

Reply:
Filing a bug sounds good.

Reply:
Done.